.travis.yml

```diff
@@ -41,8 +39,6 @@ jobs:
       script:
         - npm run test
         - npm run build
-        - npm run start &
-        - npx playwright test
```

Comment:
Are the Playwright tests removed from the .travis.yml because the e2e tests are run as part of a GitHub Action instead?

Reply:
Correct!  We are migrating to GitHub Actions